tensorflow/core/common_runtime/mkl_layout_pass_test.cc

```diff
@@ -5134,6 +5135,20 @@ TEST_F(MklLayoutPassTest, BatchMatMulV2_Positive) {
             "A(Input);B(Input);C(_MklBatchMatMulV2)|A->C;B->C:1");
 }
 
+TEST_F(MklLayoutPassTest, Einsum_Positive) {
+  InitGraph(
+      "node{ name: 'B' op: 'Float32InputList'"
+      " attr { key: 'N'                value { i: 2 } }}"
+      "node{ name: 'C' op: 'Einsum'"
+      " attr { key: 'equation'     value { s: '->' }}"
+      " attr{ key: 'N'                value { i: 2 } }"
+      " attr{ key: 'T'      value { type: DT_FLOAT } }"
```

Comment:
Nit: Can we add a space between `node/attr` and `{` and vertical-align the `value`s?
```suggestion
      "node { name: 'B' op: 'Float32InputList'"
      " attr { key: 'N'            value { i: 2 } }}"
      "node { name: 'C' op: 'Einsum'"
      " attr { key: 'equation'     value { s: '->' }}"
      " attr { key: 'N'            value { i: 2 } }"
      " attr { key: 'T'            value { type: DT_FLOAT } }"
```